developer/Makefile

```diff
@@ -29,6 +29,10 @@ run-catalog:
 	cd ..; packer validate -var-file="developer/variables-catalog-version.pkrvars.hcl" developer/examples/build.vpc.catalogimage-version.pkr.hcl
 	cd ..; packer build -var-file="developer/variables-catalog-version.pkrvars.hcl" developer/examples/build.vpc.catalogimage-version.pkr.hcl
 	
+run-catalog-version:
```

Comment:
Could we also have a command to support `catalog-offering-crn` ?
Also what would be the use case of running - `run-catalog`?